### DISCUSSION TITLE
Menu taking extra space ?

### DISCUSSION THREAD

**Discussion by BinarySenseiii** [chakra-ui/chakra-ui#7429] [General]
![image](https://user-images.githubusercontent.com/68985954/223664050-744c4392-166f-464c-8722-14109baf49a7.png)
i tried like this    
` <Box
                                position="absolute"
                                bottom={0}
                                right={0}
                                cursor="pointer"
                              >
                                <Menu gutter={0}>
                                  <MenuButton as="button">
                                    <BsThreeDotsVertical />
                                  </MenuButton>
                                  <MenuList minW={0}>
                                    <MenuItem
                                      _hover={{
                                        backgroundColor: 'accent.100',
                                        color: 'white',
                                      }}
                                    >
                                      <Flex alignItems="center" gap="5px">
                                        <BsFillPencilFill fontSize={12} />
                                        <Text>Edit</Text>
                                      </Flex>
                                    </MenuItem>
                                    <MenuItem
                                      _hover={{
                                        backgroundColor: 'accent.100',
                                        color: 'white',
                                      }}
                                    >
                                      <Flex alignItems="center" gap="5px">
                                        <FaTrashAlt fontSize={12} />
                                        <Text>Delete</Text>
                                      </Flex>
                                    </MenuItem>
                                  </MenuList>
                                </Menu>
                              </Box>`
                              
 but bcz it's taking extra space n m facing overflow issues is there a workaround to fix it? its happening only before click after click its fine

**Comment by MarcMendoza** [comment]
Just ran into this issue and was able to fix it by wrapping `<MenuList>...</MenuList>` with a `<Portal>` component. This moves MenuList in the DOM hierarchy so it won't be affected by the parent styles.

Chakra UI Portal Component Docs: https://chakra-ui.com/docs/components/portal/usage
